modules/costs/spec/requests/api/cost_entries/cost_entry_resource_spec.rb

```diff
@@ -52,44 +56,44 @@
   describe "cost_entries/:id" do
     let(:get_path) { api_v3_paths.cost_entry cost_entry.id }
 
-    context "user can see cost entries" do
-      context "valid id" do
+    context "when user can see cost entries" do
+      context "with a valid id" do
         it "returns HTTP 200" do
-          expect(response.status).to be(200)
+          expect(response).to have_http_status(200)
         end
       end
 
-      context "invalid id" do
+      context "with an invalid id" do
         let(:get_path) { api_v3_paths.cost_type "bogus" }
 
         it_behaves_like "not found"
       end
     end
 
-    context "user can only see own cost entries" do
+    context "when user can only see own cost entries" do
       let(:permissions) { [:view_own_cost_entries] }
 
-      context "cost entry is not his own" do
+      context "when ost entry is not his own" do
         it_behaves_like "error response",
                         403,
                         "MissingPermission",
                         I18n.t("api_v3.errors.code_403")
       end
 
-      context "cost entry is his own" do
-        let(:cost_entry) { create(:cost_entry, project:, user: current_user) }
+      context "when cost entry is his own" do
```

Comment:
we should probably use gender-neutral language here:

```suggestion
      context "when cost entry is their own" do
```